examples/deploy_nsx_cluster/README.md

```diff
@@ -16,5 +16,6 @@ to match your needs and run the playbooks in the order listed.
 
 Validated against:
 * NSX-T 2.4 GA
+* NSX-T 2.5 GA
```

Comment:
Please add NSX-T 3.0 GA as well to this list. Or I can add it later too. Not a big deal.